features/facebookImport/src/model/importers/utils/importer-util.js

```diff
@@ -21,17 +21,22 @@ async function readJSONFile(relativeFilePath, zipFile) {
     return readFullPathJSONFile(entry);
 }
 
+function decode(s) {
+    let d = new TextDecoder();
+    let a = s.split("").map((r) => r.charCodeAt());
+    return d.decode(new Uint8Array(a));
+}
+
 async function readFullPathJSONFile(entry) {
     const rawContent = await entry.getContent();
-    const fileContent = new TextDecoder("utf-8").decode(rawContent);
-
+    const fileContent = new TextDecoder().decode(rawContent);
     if (!fileContent) {
         throw new MissingContentImportException(entry._id);
     }
 
     return JSON.parse(fileContent, (key, value) => {
         if (typeof value === "string") {
-            return decodeURIComponent(escape(value));
+            return decodeURIComponent(decode(value));
```

Comment:
should we catch any errors of the `decodeURIComponent` here?
as there are cases it can throw an `URIError` according to the documentation

Reply:
We can log the error, and then it will appear in the logs that test users send us.

Reply:
Probably out of scope, but we could definitely do it in another PR.